### PR TITLE
fix raid setup and disable cpuscaling setup

### PR DESCRIPTION
**Pull request by amoskong**
We just added a second disks to test vms for raid setup,
this patch adjusted raid option and disable cpuscaling
setup for a known issue [1] will cause full setup fails.

[1] https://github.com/scylladb/scylla/issues/2051

### CODE REVIEW

**Comment by amoskong** [comment]
Jenkins test jobs:
- Centos7 without 2nd disk: [good ](http://jenkins.cloudius-systems.com/job/scylla-1.7-centos-rpm-test-amostmp/3/)
- Centos7 with 2nd disk: [good ](http://jenkins.cloudius-systems.com/job/scylla-1.7-centos-rpm-test-amostmp/4/)
- Ubuntu14 without 2nd disk: [good](http://jenkins.cloudius-systems.com:8080/job/scylla-1.7-ubuntu-deb-test-amostmp/3/)

File: scylla-artifacts.py [hunk 1]
```diff
@@ -275,7 +275,20 @@ def run(self):
...
+        devlist = result.stdout.split()
+        if devlist:
+            setup_cmd += ' --disks %s' % devlist[-1]
+        else:
+            setup_cmd += ' --no-raid-setup'
+
+        # disable cpuscaling setup for known issue:
+        # https://github.com/scylladb/scylla/issues/2051
+        setup_cmd += ' --no-cpuscaling-setup'
+        process.run(setup_cmd, shell=True)
```
Comment (lmr) [inline scylla-artifacts.py]: Nice

File: scylla-artifacts.py [hunk 2]
```diff
@@ -275,7 +275,20 @@ def run(self):
...
                          '(see logs for details)' % os.path.basename(pkg))
                 raise InstallPackageError(e_msg)
-        process.run('/usr/lib/scylla/scylla_io_setup', shell=True)
+
+        # enable raid setup when second disk exists
+        setup_cmd = 'sudo /usr/lib/scylla/scylla_setup --nic eth0'
+        result = process.run('ls /dev/[hvs]db', shell=True, ignore_status=True)
+        devlist = result.stdout.split()
+        if devlist:
+            setup_cmd += ' --disks %s' % devlist[-1]
```
Comment (lmr) [inline scylla-artifacts.py]: I'm surprised there wasn't a previous call to `scylla_setup` here.

Comment (amoskong) [inline scylla-artifacts.py]: Thanks for your time!

**Pull request merged**